oarepo_ui/templates/components/datafields/IBaseField.jinja

```diff
@@ -1,4 +1,4 @@
-{#def label, label_class="", data_class="" #}
+{#def label, label_class="ui cf-field-label", data_class="ui cf-field-data" #}
```

Comment:
would it be possible to name the component without the cf- prefix? The reason is that the base field can be used outside the custom fields as well